packages/@glimmer/core/test/utils/dom.ts

```diff
@@ -0,0 +1,11 @@
+export function find(selector: string): Element {
+  const element = document.getElementById('qunit-fixture')!;
+  return element.querySelector(selector)!;
+}
+
+export function click(selector: string): void {
+  const element = document.getElementById('qunit-fixture')!;
+  const clickable = element.querySelector(selector)! as HTMLButtonElement;
+
+  clickable.click();
```

Comment:
Should be using dispatch event here probably. Seems fine for now, it just reminds me that we really need to genericize the `@ember/test-helpers` DOM utilities to work without Ember / Glimmer.